core/block/export/export.go

```diff
@@ -1037,7 +1029,49 @@ func (e *export) createProfileFile(spaceID string, wr writer) error {
 	return nil
 }
 
-func (e *export) objectValid(sbType smartblock.SmartBlockType, info *model.ObjectInfo, includeArchived bool, isProtobuf bool) bool {
+func makeMarkdownName(s *state.State, wr writer, docID string, conv converter.Converter, spaceId string) string {
+	name := pbtypes.GetString(s.Details(), bundle.RelationKeyName.String())
+	if name == "" {
+		name = s.Snippet()
+	}
+	path := ""
+	// space can be empty in case user want to export all spaces
+	if spaceId == "" {
+		spaceId := pbtypes.GetString(s.LocalDetails(), bundle.RelationKeySpaceId.String())
+		path = filepath.Join(spaceDirectory, spaceId)
+	}
+	return wr.Namer().Get(path, docID, name, conv.Ext())
```

Comment:
As far as I see, it already implemented as makeFileName or I don't understand something? 🧐
`wr.Namer().Get(path, docID, name, conv.Ext())`

Reply:
I mean use extension string instead of converter.Converter
```
func (e *export) makeMarkdownName(s *state.State, wr writer, docID string, conv converter.Converter, spaceId string) string {
```

Reply:
Fixed